api-report/container-runtime.api.md

```diff
@@ -50,6 +52,8 @@ import { ITelemetryContext } from '@fluidframework/runtime-definitions';
 import { ITelemetryLogger } from '@fluidframework/common-definitions';
 import { MessageType } from '@fluidframework/protocol-definitions';
 import { NamedFluidDataStoreRegistryEntries } from '@fluidframework/runtime-definitions';
+import { SerializedIdCompressorWithNoSession as SerializedIdCompressorWithNoSession_2 } from '@fluidframework/runtime-definitions';
```

Comment:
rename issue?

Reply:
Did some clean-up with the exports and also have removed some identifiers that were in duplicate places (and possibly causing this).